### PR TITLE
Bionic conflicts and minor fixes

### PR DESCRIPTION
**Pull request by anoobindisguise**
#### Summary
Balance "More bionic/mutation conflicts, bionic fixes"

#### Purpose of change
I like how bionics conflict with mutations and wanted to expand on this. Also fixed some bugs and typos.

#### Describe the solution
- Titanium Skeletal Bracing conflicts with most mutations that would alter your skeleton aside from hollow bones. If you became an amorphous slime or had tentacle limbs, there would be no skeleton to brace. 
- Expanded Digestive System now cancels all digestive mutation and traits, such as weak or strong stomach, nausea/vomit. eatpoison/eatdead, carnivore/herbivore, alcohol metabolism, and intolerances/allergies. Because it is replacing your intestines and stomachs it would no longer be possible for those organs to mutate, and food allergies would no longer apply.
- Telescopic Eyes conflicts with gastropod eyestalks.
- Alloy CBMs have had their names reverted from "Intradermal Alloy" to "Alloy Plating" as they were before. In addition, torso alloy plating now has 1 encumbrance instead of 3. All other body parts had 1 encumbrance, the torso having 3 encumbrance was a mistake on my part because the CBMs were originally going to provide much more defense in my earlier PR.
- Voice Remodulator cancels the Croaking Voice mutation of the Frog line.
- Joint Torsion Ratchet conflicts with mutations radically altering arms and legs (tentacles, amorphous, gastropod foot) for the same reasons as skeletal bracing.
- Bionic Claws conflict with mutations that radically alter your hands (tentacles, pseudopods) and claws mutations. Does not conflict with paws, as the claws could still feasibly fit within your 5 "fingers". Other hand bionics (fingerhack, fingerpick, finger lighter, finger laser, toolset, needledrip) currently do not conflict. Subject to change.

#### Describe alternatives you've considered
- Remove encumbrance from Alloy CBMs entirely, I'm not sure if it needs to be there.
- Adjust slot usage of CBMs. If Expanded Digestive replaces the user's organs, it frees up a lot of space and thus should use fewer slots. Power Storage CBMs should also use slots now that they can be installed and uninstalled freely. However I'm not sure if I should touch slot usage at the moment because it is due to be overhauled anyways.

### CODE REVIEW

**Comment by PatrikLundell** [comment]
Alloy plating ought to affect encumbrance only if they have a significant weight (in particular for the torso to be affected), or they restrict the movement range or sensory sensitivity of the body part somewhat.

**Comment by scavtoons** [comment]
On a related issue, what if mutations that affect size also affect the number of cbm slots you can carry? At least on the torso area.

File: data/json/bionics.json [hunk 1]
```diff
@@ -362,29 +361,29 @@
...
       "BOVINE_BULK",
       "BOVINE_BULK2",
       "FAT",
       "SLIME_SPRAY",
       "GASTROPOD_FOOT",
       "GASTROPOD_BALANCE",
       "MUCUS_SECRETION",
-      "MUCUS_SECRETION2"
+      "MUCUS_SECRETION2",
+      "RABBIT_FEET"
```
Comment (kevingranade) [inline data/json/bionics.json:372]:
```suggestion
      "MUCUS_SECRETION2"
```

**Comment by anoobindisguise** [comment]
@kevingranade may I ask why you removed rabbit_feet from legs plating? It affects the foot, it should probably conflict.

**Comment by kevingranade** [comment]
It was failing to load because there was already an instance of rabbit feet in the list.

**Comment by anoobindisguise** [comment]
> It was failing to load because there was already an instance of rabbit feet in the list.

Lol I'm blind. OK. Any other conflicts you can think of? Fingerhack, pick, lighter, tools, claws, needle CBMs should probably conflict with arm tentacles now that I think about it.

**Pull request merged**